deploy/olm-catalog/kubevirt-hyperconverged/0.0.2/metrics-aggregation.crd.yaml

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
nit: remove this empty line?

Reply:
This PR is not for CSV templates fixing, I'd rather keep this PR to the point, if that's fine